mobile_back_tflite/cpp/backend_tflite/stable_diffusion_pipeline.cc

```diff
@@ -64,6 +64,15 @@ mlperf_backend_ptr_t StableDiffusionPipeline::backend_create(
   SDBackendData* backend_data = new SDBackendData();
   backendExists = true;
 
+  for (int i = 0; i < configs->count; ++i) {
+    if (strcmp(configs->keys[i], "stable_diffusion_seed") == 0) {
+      backend_data->seed = atoi(configs->values[i]);
+    }
+    if (strcmp(configs->keys[i], "stable_diffusion_num_steps") == 0) {
+      backend_data->num_steps = atoi(configs->values[i]);
+    }
+  }
+
```

Comment:
The backend can read them from `mlperf_backend_configuration_t` in `mlperf_backend_create`.